src/styles.css

```
@@ -707,3 +707,130 @@ footer a:hover{text-decoration:underline}
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 2.5px;
+  background: linear-gradient(90deg, var(--orange), #FBBC04);
+  opacity: 0;
+  transition: opacity .2s;
+}
+.recent-card:hover {
+  border-color: var(--orange-border);
+  box-shadow: 0 6px 24px rgba(244, 107, 14, .12);
```

Stylelint (17.11.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)